From 008f569fc0c427fcb798a1ef306c1f780450f11b Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 7 Oct 2015 21:07:49 -0700
Subject: [PATCH 3/6] Add initial systemd service file

Should probably make a template .in file for this one as well, similar
to the DBus service.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 Makefile.am           | 1 +
 src/pacrunner.service | 9 +++++++++
 2 files changed, 10 insertions(+)
 create mode 100644 src/pacrunner.service

diff --git a/Makefile.am b/Makefile.am
index 4cd0971..e520416 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -182,6 +182,7 @@ endif
 CLEANFILES = src/builtin.h
 
 EXTRA_DIST = src/genbuiltin src/pacrunner.conf $(test_scripts) \
+			src/pacrunner.service \
 			doc/architecture.txt doc/libproxy.txt \
 			doc/manager-api.txt doc/client-api.txt \
 			doc/manual-configuration.txt
diff --git a/src/pacrunner.service b/src/pacrunner.service
new file mode 100644
index 0000000..6445215
--- /dev/null
+++ b/src/pacrunner.service
@@ -0,0 +1,9 @@
+[Unit]
+Description=Proxy AutoConfig runner service
+
+[Service]
+ExecStart=/usr/sbin/pacrunner -n
+User=pacrunner
+
+[Install]
+WantedBy=multi-user.target
-- 
2.13.3

